docs/mobile/events.md

```diff
@@ -21,46 +21,59 @@ application lifecycle. This event is meant to be used in conjunction with
 `os.name` [resource semantic convention](/docs/resource/os.md) to identify the
 mobile operating system (e.g. Android, iOS).
 
-### iOS
+### Event name
 
-<!-- semconv ios.lifecycle.events -->
+<!-- semconv device.app.lifecycle(full) -->
 The event name MUST be `device.app.lifecycle`.
 
-| Attribute  | Type | Description  | Examples  | Requirement Level |
-|---|---|---|---|---|
-| `ios.state` | string | This attribute represents the state the application has transitioned into at the occurrence of the event. [1] | `active` | Required |
-
-**[1]:** The iOS lifecycle states are defined in the [UIApplicationDelegate documentation](https://developer.apple.com/documentation/uikit/uiapplicationdelegate#1656902), and from which the `OS terminology` column values are derived.
-
-`ios.state` MUST be one of the following:
-
-| Value  | Description |
-|---|---|
-| `active` | The app has become `active`. Associated with UIKit notification `applicationDidBecomeActive`. |
-| `inactive` | The app is now `inactive`. Associated with UIKit notification `applicationWillResignActive`. |
-| `background` | The app is now in the background. This value is associated with UIKit notification `applicationDidEnterBackground`. |
-| `foreground` | The app is now in the foreground. This value is associated with UIKit notification `applicationWillEnterForeground`. |
-| `terminate` | The app is about to terminate. Associated with UIKit notification `applicationWillTerminate`. |
 <!-- endsemconv -->
 
-### Android
+### Event payload (Log Body)
 
-<!-- semconv android.lifecycle.events -->
-The event name MUST be `device.app.lifecycle`.
+Payload attributes MUST be used to describe the state of the application at the
+time of the event. The following table describes the payload attributes that MUST
+be used to describe the state of the application at the time of the event.
 
+The `android.state` and `ios.state` attributes are mutually exclusive and MUST
```

Comment:
There are some native mobile apps where there are two platforms at play. For instance, Unity apps are written in a cross-platform way and can be deployed on both iOS and Android, in which case we want to convey that it's a Unity app and the event is firing in a specific platform. 

If we have this mutual exclusivity, how can we represent that? I can see at least two options:

- Allow duplication of states, and rely on the fact that an event can't both be iOS and Android at the same time, but could potentially be one of each, and then also be Unity, React Native, Kotlin Multiplatform, etc.
- Have a separate (optional) attribute that represents the framework used to generate the app, and keep the actual OS state mutually exclusive.

BTW, this may be beyond the scope of this change - if so, I can submit another pull request to add this.